### ISSUE TITLE
Overflow hidden is not set to the html tag if the modal is initially set to open

### ISSUE THREAD

**Issue by iafiawik** [pradel/react-responsive-modal#18]
### Expected behaviour
Irrespective of when and how the `Modal` component is being opened it should disable scroll of its topmost parent.

### Actual behaviour
If a `Modal` component is receiving `{open: true}` as its initial props it will not set `overflow: hidden` etc. to the top `<html>` tag like it does when it is opened by a button click.

### How to reproduce the behaviour
```js
import React from "react";
import createReactClass from "create-react-class";
import Modal from "react-responsive-modal";

var Component = createReactClass({
  getInitialState: function() {
    return {
      isOpen: true
    };
  },

  _onDialogCloseHandler: function() {
    this.setState({
      isOpen: false
    });
  },

  render: function() {
    const { isOpen } = this.state;

    return (
      <Modal open={isOpen} onClose={this._onDialogCloseHandler}>
        <div>Content</div>
      </Modal>
    );
  }
});
```

### Versions
`react@16.0.0`
`react-responsive-modal@1.9.0`

**Comment by pradel** [comment]
I never faced this issue yet but you are right I will push a fix soon!

**Comment by pradel** [comment]
Published as 1.9.3
